docs/api/authentication.md

```diff
@@ -1,44 +1,42 @@
 # Service accounts authentication
 
 !!! caution alert alert-warning "Deprecation notice"
-Starting with version 3, PMM no longer uses API keys as the primary method for controlling access to the PMM Server components and resources. Instead, PMM is now leveraging Grafana service accounts, which have limited scopes and offer enhanced security compared to API keys.
+    Starting with version 3, PMM no longer uses API keys as the primary method for controlling access to the PMM Server components and resources. Instead, PMM is now leveraging Grafana service accounts, which have limited scopes and offer enhanced security compared to API keys.
 
-**Automatic migration of API keys**
-When you install PMM 3.x, any existing API keys will be seamlessly converted to service accounts with corresponding service tokens.
+### Automatic migration of API keys
+
+When you install PMM v3.x, any existing API keys will be seamlessly converted to service accounts with corresponding service tokens.
 
 Service accounts in PMM provide a secure and efficient way to manage access to the PMM Server and its resources. They serve as a replacement for the basic authentication and API keys used in previous versions of PMM (v.2 and earlier).
 
 With service accounts, you can:
 
-- Control access to PMM Server components and resources.
-- Define granular permissions for various actions.
-- Create and manage multiple access tokens for a single service account.
+- control access to PMM Server components and resources.
+- define granular permissions for various actions.
+- create and manage multiple access tokens for a single service account.
 
 Creating multiple tokens for the same service account is beneficial in the following scenarios:
 
 - when multiple applications require the same permissions but need to be audited or managed separately. By assigning each application its own token, you can track and control their actions individually.
-when a token becomes compromised and needs to be replaced. Instead of revoking the entire service account, you can rotate or replace the affected token without disrupting other applications using the same service account.
+- when a token becomes compromised and needs to be replaced. Instead of revoking the entire service account, you can rotate or replace the affected token without disrupting other applications using the same service account.
 - when you want to implement token lifecycle management. You can set expiration dates for individual tokens, ensuring that they are regularly rotated and reducing the risk of unauthorized access.
 
+## Generate a service account and token
 
-##  Generate a service account and token 
-
-PMM uses the Grafana service account tokens for authentication. A token is a generated random string that acts as an alternative to a API passwords.
+PMM uses the Grafana service account tokens for authentication. A token is a generated random string that acts as an alternative to API passwords.
```

Comment:
API keys or basic auth passwords

Reply:
PMM uses Grafana service account tokens for authentication. These tokens are randomly generated strings that serve as alternatives to API keys or basic authentication passwords.